pybaseball/lahman.py

```diff
@@ -26,15 +29,20 @@ def download_lahman():
     # download entire lahman db to present working directory
     z = get_lahman_zip()
     if z is not None:
-        z.extractall()
+        z.extractall(caching.cache_config.cache_directory)
         z = get_lahman_zip()
         # this way we'll now start using the extracted zip directory
         # instead of the session ZipFile object
 
 def _get_file(tablename: str, quotechar: str = "'") -> pd.DataFrame:
     z = get_lahman_zip()
     f = f'{base_string}/{tablename}'
-    data = pd.read_csv(f if z is None else z.open(f), header=0, sep=',', quotechar=quotechar)
+    data = pd.read_csv(
+        f"{path.join(caching.cache_config.cache_directory, f)}" if z is None else z.open(f),
```

Comment:
Do this on `base_string` instead I think

Reply:
Due to the fact that the base string is the relative path in the zip file AND on the local file system, I don't believe you can.